tests/updateArticle.test.js

```diff
@@ -121,7 +122,8 @@ describe('testing for the article update controller', () => {
       .set('authorization', UserToken)
       .send({
         body: '',
-        tags: ['Laravel', 'php', 'IOT', 'iot2']
+        tags: ['Laravel', 'php', 'IOT', 'iot2'],
+        coverImage: 'http://localhost:3000/ver.jpg'
```

Comment:
@truestbyheart only the author of a draft article should be able to view/update it. Is there a check for this?

Reply:
I didn't implement that let work on it.